packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -39,14 +39,23 @@ const MAX_ATTEMPTS = 20
  * ```
  *
  * @param this - Client.
- * @param wallet - An existing XRPL Wallet to fund, if undefined, a new Wallet will be created.
+ * @param wallet - An existing XRPL Wallet to fund. If undefined or null, a new Wallet will be created.
+ * @param options - See below.
+ * @param options.faucetHost - A custom host for a faucet server. On devnet and
+ * testnet, `fundWallet` will attempt to determine the correct server
+ * automatically. In other environments, or if you would like to customize the
+ * faucet host in devnet or testnet, you should provide the host using this
+ * option.
```

Comment:
It might be nice to include an example to clarify if the port is needed, if https://... is needed, etc.